api/lib/JSONValidator.js

```diff
@@ -0,0 +1,26 @@
+const Ajv = require('ajv');
+const ajv = new Ajv({allErrors: true});
+
+const Schemas = require('./schemas');
+const errorMessages = require('./error-messages');
+
+function JSON(){
+    this.validSchemas = Object.keys(Schemas);
+}
+
+JSON.prototype.isValid = function(document, schema) {
+    return new Promise((res, rej) => {
```

Comment:
This function is not async, so there's no need to use promises.

Reply:
So, just to explain this better, this function should return true if `doc` follows schema `schema`, false otherwise.

I liked that you checked if `schema` was a valid, defined, schema, I think we should maintain that but instead of returning a rej(), we should throw the `Error`

Reply:
Ok I'll fix that!